src/mysql_protocol.erl

```diff
@@ -203,6 +203,26 @@ execute(#prepared{statement_id = Id, param_count = ParamCount}, ParamValues,
 fetch_execute_response(SockModule, Socket, Timeout) ->
     fetch_response(SockModule, Socket, Timeout, binary, []).
 
+-spec change_user(atom(), term(), iodata(), iodata(), binary(), iodata()) ->
+    #ok{} | #error{}.
+change_user(SockMod, Socket, Username, Password, Salt, Database) ->
```

Comment:
Database can also be undefined (in the spec).

Reply:
Ah, yes, right. Will fix.

Reply:
Fixed.